### DISCUSSION TITLE
How to call javascript functions

### DISCUSSION THREAD

**Discussion by felipetesc** [moonbitlang/moon#539] [Q&A] [answered]
Hello. Is it possible to call JS functions ?

**Comment by Young-Flash** [comment, marked as answer]
hi, you can use inline js for js backend
https://github.com/moonbitlang/x/blob/9f5594cd7697b5cb4214dd176462929d4476e5aa/fs/internal/ffi/fs_js.mbt#L21-L26

**Reply by felipetesc** [reply]
Thanks.
